### DISCUSSION TITLE
Generate a logfile? (Need to debug crashing).

### DISCUSSION THREAD

**Discussion by midijohnny** [sqlitebrowser/sqlitebrowser#3782] [Q&A]
I love the product - very useful and productive.
However: I do experience a few 'random' looking crashes - and not all that infrequently. 
I don't know how to investigate these crashes: and since there doesn't appear to be an obvious pattern in how to re-create a crash : I was hoping there was a way of just leaving DEBUG logs on , so I can check in the event of a crash.

Is that do-able?

**Comment by justinclift** [comment]
We've been fixing a few crashing bugs in the last few days, including one in the new 3.13.1 release.

Would you be ok to install the latest "continuous" build (it's generated from our very latest development code) and see if the crashes still happen for you?  https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/continuous

**Comment by midijohnny** [comment]
Thanks! Will do so : for reference - I will use this version:

```
DB Browser for SQLite Version 3.13.99 (Oct 19 2024)
Last commit hash when built: 51c37de

Built for x86_64-little_endian-lp64, running on x86_64
Qt Version 5.15.13
SQLite Version 3.46.1.
```

**Comment by mgrojo** [comment]
Are you on Windows?

https://github.com/sqlitebrowser/sqlitebrowser/wiki/Debugging-a-crash-on-Windows

**Comment by midijohnny** [comment]
I am (mainly) using Linux - but using both Windows and Linux.
After using "DB Browser for SQLite Version 3.13.99 (Oct 19 2024)" for a few weeks now - I haven't seen a single crash ! 
Thanks for the advice!

BTW: small issue (I'll log separately if need be) - the 'CTRL-S' (to save) doesn't appear to work now in the "Execute SQL" tab.

**Reply by mgrojo** [reply]
> BTW: small issue (I'll log separately if need be) - the 'CTRL-S' (to save) doesn't appear to work now in the "Execute SQL" tab.

It never worked in that way. See #2798
